[DE Team][8.15][Serverless] Bulk-update a rule's custom highlighted fields (#5460)
elastic/security-docs: docs/detections/rules-ui-create.asciidoc

.. *False positive examples* (optional): List of common scenarios that may produce
false-positive alerts.
.. *MITRE ATT&CK^TM^ threats* (optional): Add relevant https://attack.mitre.org/[MITRE] framework tactics, techniques, and subtechniques.

> .. *Custom highlighted fields* (optional): Specify one or more highlighted fields for personalized alert investigation flows. You can choose any fields that are available in the <<update-sec-indices,default {elastic-sec} indices>>. There is no limit to the number of fields that you can add. 

Reviewer: This is incorrect. Here we will use fields from indices selected for the rule and only in case of bulk action we will use default indices. So, to summarize:
- Edit rule settings > About section: we use indices selected in the rule settings
- Bulk action: we use default indices from advanced security solution settings

That is why we also should not put `There is no limit to the number of fields that you can add.` here.

Author: Gotcha - makes sense and thanks for clarifying this. Here's the corrected version -

```suggestion
.. *Custom highlighted fields* (optional): Specify one or more highlighted fields for personalized alert investigation flows. You can choose any fields that are available in the indices you selected for the rule's data source.  
```

.. *False positive examples* (optional): List of common scenarios that may produce
false-positive alerts.
.. *MITRE ATT&CK^TM^ threats* (optional): Add relevant https://attack.mitre.org/[MITRE] framework tactics, techniques, and subtechniques.

> .. *Custom highlighted fields* (optional): Specify one or more highlighted fields for personalized alert investigation flows. You can choose any fields that are available in the indices you selected for the rule's data source.  

Reviewer 2: Optional Nit: "Personalized" sounds kinda like it could be tailored to an individual user, but custom highlighted fields are probably system-wide. "Customized" seems more accurate but basically repeats "custom" -- maybe "unique"?

Author: Unique works 👍🏼 

```suggestion
.. *Custom highlighted fields* (optional): Specify one or more highlighted fields for unique alert investigation flows. You can choose any fields that are available in the indices you selected for the rule's data source.  
```
